src/View/User/Includes/_inscription_form_course.html.twig

```diff
@@ -83,9 +83,9 @@
                 <div class="mb-3 field">
                     <label for="phone" class="form-label">
                         <span class="text-danger">*</span>Téléphone :</label>
-                    <input type="text" class="form-control" id="phone" name="phone" placeholder="06.00.00.00.00." required value="{{ course.phone }}" aria-required="true">
+                    <input type="text" class="form-control" id="phone" name="phone" placeholder="06.00.00.00.00." required value="{{ course.phone_number }}" aria-required="true">
```

Comment:
there's a type="tel" in html, that is basically a type="text" but that opens an appropriate "only numbers" keyboard for phones and tablets ;)

Reply:
type="tel" seems to return strings too, change text to tel will not change anything

Reply:
as explained above, it does change the UX for phone users, but ok